.tools/readmes/scanner.py

```diff
@@ -43,6 +43,9 @@ def load_crosses(self):
         self.doc_gen.process_metadata(
             self.doc_gen.root / ".doc_gen" / "metadata" / "cross_metadata.yaml"
         )
+        self.doc_gen.process_metadata(
+                    self.doc_gen.root / ".doc_gen" / "metadata" / "tributary-lite_metadata.yaml"
+                )
 
```

Comment:
You shouldn't need this anymore, the scanner should pick up this file and handle it appropriately now.